src/api_routes_http.rs

```diff
@@ -146,7 +147,11 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           // Admin Actions
           .route("", web::post().to(create_site))
           .route("", web::put().to(update_site))
-          .route("/block", web::post().to(block_instance)),
+          .route("/block", web::post().to(block_instance))
+          // URL blocklist
+          .route("/url_blocks", web::get().to(list_url_blocks))
+          .route("/url_blocks", web::put().to(add_url_block))
+          .route("/url_blocks/remove", web::put().to(remove_url_block)),
```

Comment:
I think generally `POST` is used for remove/delete endpoints elsewhere in the API, might be nice to use it here as well for uniformity

Reply:
Sorted.